src/Peer.proto

```diff
@@ -0,0 +1,50 @@
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Network.Protocol.Protobuffs <https://github.com/catalyst-network/protocol-protobuffs>
+*
+* Catalyst.Network.Protocol.Protobuffs is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+* 
+* Catalyst.Network.Protocol.Protobuffs is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+* 
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Network.Protocol.Protobuffs If not, see <https://www.gnu.org/licenses/>.
+*/
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package Catalyst.Protocol.Peer;
+
+import "Cryptography.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+
+/**
+* Aggregated size of the fields in PeerId should be 42 bytes as specified in
+* https://github.com/catalyst-network/protocol-blueprint/blob/master/PeerProtocol.md
+*/
+message PeerId {
+    bytes protocol_version = 1; //2 bytes
+    bytes ip = 2; // 16 bytes
+    uint32 port = 3; // 2 bytes
+    Cryptography.PublicKey public_key = 4; // Peers public key, node operators are encouraged to use the same, public key across all their nodes.
+}
+
+message PeerInfo {
+    PeerId peer_id = 1; // is peerId of network
+    int32 reputation = 2; // reputation of peer
```

Comment:
Please use sint32
Check the proto docs:
https://developers.google.com/protocol-buffers/docs/proto

```
Uses variable-length encoding. Inefficient for encoding negative numbers – if your field is likely to have negative values, use sint32 instead.
--
```